scalar.c

```diff
@@ -230,6 +230,31 @@ static int set_recommended_config(int reconfigure)
 				     fsmonitor.key, fsmonitor.value);
 	}
 
+	/*
+	 * Set HTTP/1.1 for Azure DevOps URLs
+	 */
+	if (!git_config_get_string("remote.origin.url", &value)) {
+		if (strstr(value, "dev.azure.com") || 
+		    strstr(value, "visualstudio.com")) {
+			struct strbuf key = STRBUF_INIT;
+			strbuf_addf(&key, "http.%s.version", value);
+			
+			if (reconfigure || git_config_get_string(key.buf, &value)) {
```

Comment:
You need to call `free(value)` before this call. Ideally, you'd even call `FREE_AND_NULL(value)` just after using the value in that `strbuf_addf()` call.